### PR TITLE
Added flowLog to NetworkWatcher swagger + examples

### PR DESCRIPTION
**Pull request by irrogozh**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by lirenhe — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json [hunk 1]
```diff
@@ -2727,6 +2970,81 @@
         }
       }
     },
+    "FlowLogListResult": {
+      "properties": {
+        "value": {
```
Comment (ryansbenson) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json]: Include 'nextLink' even if you just set it to null. This allows you to add it later without an API breaking change.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json [hunk 2]
```diff
@@ -2024,6 +2024,249 @@
...
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./networkWatcher.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```
Comment (ryansbenson) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json]: Set this to 'nextLink'.

**Review by ryansbenson — COMMENTED** [review]
Just a minor change.

**Comment by sanjaiganesh** [comment]
@irrogozh please address the comments.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json (hunk 1 above)
Comment (irrogozh) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json (hunk 2 above)
Comment (irrogozh) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json]: Done

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Comment by lirenhe** [comment]
@irrogozh, could you take a look at the network validation issue?
Is it the same root cause of  https://github.com/Azure/azure-rest-api-specs/pull/8042

**Comment by number213** [comment]
@lirenhe this PR is good to go

**Comment by MikhailTryakhov** [comment]
@lirenhe  can you please help to merge this change?

**Pull request merged**